frontend/src/components/UserSummaryCard/constants.ts

```diff
@@ -0,0 +1 @@
+export const DEFAULT_COL_SIZE = "sm:col-12 md:col-4 lg:col-1";
```

Comment:
You’re right, it could be in the component itself. However, I’m following this structure to keep the code organized and avoid spaghetti code—it’s just a convention, and having an extra file doesn’t hurt. It makes changes easier without needing to search through a long file. Initially, I also considered separating the styles into styles.scss under the component folder, but since VSCode offers great support for in-component styling, I decided to keep it there.

**ComponentFolder:**
 - constants.ts: For constants definitions
 - definitions.ts: For types/interfaces
 - utils.ts: For any utility functions specific to this component